packages/teleport/src/services/database/databases.test.ts

```diff
@@ -0,0 +1,51 @@
+/*
+Copyright 2021 Gravitational, Inc.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+import DatabaseService from './databases';
+import api from 'teleport/services/api';
+
+test('correct formatting of database fetch response', async () => {
```

Comment:
i'd check all possible combo of protocol type and instead of matching the entire object, i'd just check the `type` field